unittests/test_pipeline.py

```diff
@@ -763,6 +763,23 @@ def validate(self):
     _run(MyTest(), *local_exec_ctx)
 
 
+def test_sourcesdir_git(local_exec_ctx):
+    @test_util.custom_prefix('unittests/resources/checks')
+    class MyTest(rfm.RunOnlyRegressionTest):
+        sourcesdir = 'https://github.com/octocat/hello-worId.git'
```

Comment:
```suggestion
        sourcesdir = 'https://github.com/octocat/hello-world.git'
```
I fine with keeping this or creating a repo under reframe-hpc. But is the capital i in the path a typo? 😆 Both repos exist but I assume the hello world is the original

Reply:
True! I think it's better to have a repo hosted in reframe-hpc so that we can control it.